src/xrGame/ai_space.h

```diff
@@ -86,6 +91,14 @@ class CAI_Space : public AISpaceBase
     IC doors::manager& doors() const;
 };
 
+// Static initializer for every translation unit
+// Need to avoid static initilization order problem
+static struct SAI_Space_Initializer
+{
+    SAI_Space_Initializer();
+    ~SAI_Space_Initializer();
+} aiSpaceInitializer;
```

Comment:
According to the code style global variable name must start with capital letters.
```suggestion
} AISpaceInitializer;
```